examples/todoapp/src/App.tsx

```diff
@@ -0,0 +1,51 @@
+import { Heading, Stack } from "@chakra-ui/react";
+import { FabrixComponent, gql } from "@fabrix-framework/fabrix";
+import { css } from "@emotion/css";
+
+const containerClassName = css`
+  padding: 15px 0;
+`;
+
+function App() {
+  return (
+    <Stack padding={10}>
+      <Heading textAlign="center" size="md">
+        Fabrix TODO app example
+      </Heading>
+      <FabrixComponent
+        containerClassName={containerClassName}
+        query={gql`
+          mutation createTodo($input: TodoInput!) {
+            addTodo(input: $input)
+              @fabrixForm(
+                input: [
+                  { field: "hasDone", config: { hidden: true } }
+                  { field: "name", config: { gridCol: 9 } }
+                  { field: "priority", config: { gridCol: 3 } }
+                ]
+              ) {
+              id
+            }
+          }
```

Comment:
[ask]
After pressing the button generated from this mutation, the task list does not change, and a React warning is displayed in the browser. Is this the expected behavior?

Reply:
Let me check that on my side...

Reply:
@takumiyoshikawa 
This is a bug that I opened at https://github.com/fabrix-framework/fabrix/issues/9.

I know this is pretty critical, but can I merge this PR at first? Merging this PR really help me debug the behaviour to tackle on the issue.